src/test/java/uk/gov/ch/pocappointmentapi/PocAppointmentApiTests.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.ch.pocappointmentapi;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class PocAppointmentApiTests {
```

Comment:
Our integration tests will eventually go here; please either move this to src/it/java and add src/it/java as an extra test source or put this in a separate submodule.

Reply:
This test file and it's location is generated by Spring-Boot. We would need extra entries in the pom to support the src/it/java source folder. I would say this is not needed for a POC.